tools/modify_readme.py

```diff
@@ -0,0 +1,15 @@
+import re
+import sys
+
+repo_name = sys.argv[1]
+branch_name = sys.argv[2]
+# Regex pattern <img\s+([^>]*?)(?<!['"])(?<!\/)src\s*=\s*(['"])(?!http|\/)(.*?)\2([^>]*?)>
```

Comment:
Using the r""  syntax would have prevented you to add this comment... and simplify the next line

Reply:
for double quotes you still need the \ so it would almost be the same